awx/ui/client/src/templates/main.js

```diff
@@ -708,15 +706,18 @@ angular.module('templates', [surveyMaker.name, jobTemplates.name, labels.name, p
                                 delete list.fields.labels;
                                 delete list.fieldActions;
                                 list.fields.name.columnClass = "col-md-8";
+                                list.fields.name.tag = i18n._('WORKFLOW');
+                                list.fields.name.showTag = "{{job_template.type === 'workflow_job_template'}}";
```

Comment:
I wonder if the `job_template` variable needs to be changed to just `template`.  I think we could achieve this by changing list.iterator to `template` and list.name to `templates`.  We'd have to update all the other places that that might impact.